base_link_frame_id: base_link
lidar_frame_id: scan
thetas_frame_id: theta_s
realsense_frame_id: realsense
lidar_x: 0.0
lidar_y: 0.0
lidar_z: 0.230
lidar_roll: 0.0
lidar_pitch: 0.0
lidar_yaw: 0.0
thetas_x: 0.0
thetas_y: 0.0
thetas_z: 0.408
thetas_roll: 0.0
thetas_pitch: 0.0
thetas_yaw: 0.0
realsense_x: 0.0
realsense_y: 0.0
realsense_z: 0.663
realsense_roll: 0.0
realsense_pitch: 0.0
realsense_yaw: 0.0
